packages/neon-api/src/index.ts

```diff
@@ -21,10 +21,14 @@ function assignSettings(
 }
 function bundle<T extends typeof _Neon>(
   neonCore: T
-): T & { api: typeof plugin } {
+): T & {
+  TransactionBuilder: TransactionBuilder;

```

Comment:
if we are using the `api` package, then all these should be under the `api` namespace.